src/extensions/default/JavaScriptCodeHints/node/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "brackets-javascript-code-hints",
+  "dependencies": {
+    "acorn": "3.3.0",
```

Comment:
I didn't see `acorn` used on node side, is it?
Also in the latest releases there were some problems, I'd upgrade it in a separate PR when things are settled down on `acorn` side.